Swat/Swat.php

```diff
@@ -96,102 +88,84 @@ public static function ngettext($singular_message, $plural_message, $number)
-
     /**
-     * Displays the methods of an object
+     * Displays the methods of an object.
      *
      * This is useful for debugging.
      *
-     * @param mixed $object the object whose methods are to be displayed.
+     * @param mixed $object the object whose methods are to be displayed
      */
     public static function displayMethods($object)
     {
-        echo sprintf(self::_('Methods for class %s:'), get_class($object));
+        echo sprintf(self::_('Methods for class %s:'), $object::class);
         echo '<ul>';
 
-        foreach (get_class_methods(get_class($object)) as $method_name) {
+        foreach (get_class_methods($object::class) as $method_name) {
             echo '<li>', $method_name, '</li>';
         }
 
         echo '</ul>';
     }
 
-    // }}}
-    // {{{ public static function displayProperties()
-
     /**
-     * Displays the properties of an object
+     * Displays the properties of an object.
      *
      * This is useful for debugging.
      *
-     * @param mixed $object the object whose properties are to be displayed.
+     * @param mixed $object the object whose properties are to be displayed
      */
     public static function displayProperties($object)
     {
-        $class = get_class($object);
+        $class = $object::class;
 
         echo sprintf(self::_('Properties for class %s:'), $class);
         echo '<ul>';
 
         foreach (get_class_vars($class) as $property_name => $value) {
-            $instance_value = $object->$property_name;
+            $instance_value = $object->{$property_name};
             echo '<li>', $property_name, ' = ', $instance_value, '</li>';
         }
 
         echo '</ul>';
     }
 
-    // }}}
-    // {{{ public static function printObject()
-
     /**
-     * Displays an object's properties and values recursively
+     * Displays an object's properties and values recursively.
      *
      * Note:
      *
      * If the object being printed is a UI object then its parent property
      * is temporarily set to null to prevent recursing up the widget tree.
      *
-     * @param mixed $object the object to display.
+     * @param mixed $object the object to display
      */
     public static function printObject($object)
     {
         echo '<pre>' . print_r($object, true) . '</pre>';
     }
 
-    // }}}
-    // {{{ public static function displayInlineJavaScript()
-
     /**
      * Displays inline JavaScript properly encapsulating the script in a CDATA
-     * section
+     * section.
      *
-     * @param string $javascript the inline JavaScript to display.
+     * @param string $javascript the inline JavaScript to display
      */
     public static function displayInlineJavaScript($javascript)
     {
         if ($javascript != '') {
             echo '<script type="text/javascript">',
-                "\n//<![CDATA[\n",
-                rtrim($javascript),
-                "\n//]]>\n</script>";
+            "\n//<![CDATA[\n",
```

Comment:
This formatting is a bit weird. Perhaps switching to concatenation `.` would improve the autoformat.